R/MeasureFairness.R

```diff
@@ -0,0 +1,54 @@
+#' @title Fairness Measure
+#' @name mlr_measures_fairness
+#'
+#' @description
+#' This measure specializes [Measure] for Fairness Measure problems:
+#' Users could use [groupwise_abs_diff()], [groupwise_diff()], or [groupwise_quotient()] to evaluate
+#' the fairness measures based on base_measures. For example, the false positive rate bias or
+#' the equalized odds ratios. More examples could viewed on mlr3fairness wikipage.
```

Comment:
Do not reference the wiki in the docs, instead either point to a vignette or to the examples section.

Reply:
Are there any vignette examples I could review?

Reply:
https://github.com/mlr-org/mcboost/tree/main/vignettes

Most of the `mlr` vignettes have been integrated in the mlr3 book, that is why mlr often does not have any.
We will at a later point perhaps also add the fairness vignettes to mrl3measures, but for now let us develop them with the package.